src/easylink/runner.py

```diff
@@ -105,3 +108,16 @@ def get_environment_args(config: Config) -> List[str]:
             "only computing_environment 'local' and 'slurm' are supported; "
             f"provided {config.computing_environment}"
         )
+
+
+def save_dag_image(snakefile, results_dir) -> None:
```

Comment:
And this will be wrapped up in a new click command?

Reply:
I have not designed anything like this, but agree it would be useful!

Reply:
Entirely separate command!